ground/gcs/src/plugins/config/configgadgetwidget.cpp

```diff
@@ -58,8 +58,11 @@ ConfigGadgetWidget::ConfigGadgetWidget(QWidget *parent) : QWidget(parent)
 {
     setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
 
+    this->setObjectName(QString("ConfigWidget"));
```

Comment:
Ack. Remember also that you can quite happily have multiple Config widgets in your layout so referring to it by class seems a much better idea.